dockerfile-prs

```diff
@@ -135,52 +142,58 @@ EXPOSE 8080
 
 FROM base AS prod-builder
 # Install development dependencies for all
-COPY package.json .
-COPY pnpm-workspace.yaml .
-COPY pnpm-lock.yaml .
 COPY tsconfig.json .
 COPY .eslintrc.js .
 COPY .pnpmfile.cjs .
-COPY packages/common/package.json ./packages/common/
-COPY packages/warehouses/package.json ./packages/warehouses/
-COPY packages/backend/package.json ./packages/backend/
-COPY packages/frontend/package.json ./packages/frontend/
 
-RUN --mount=type=cache,id=pnpm,target=/pnpm/store \
-    pnpm install --frozen-lockfile --prefer-offline
+COPY pnpm-lock.yaml pnpm-workspace.yaml package.json ./
+COPY packages/common/package.json packages/common/
```

Comment:
package.json is pretty much going to change on every PR (and when pulling master) becuase of the version